components/Card/Grid/ProductGridTab.vue

```diff
@@ -124,10 +136,21 @@ export default {
         ...mapActions('productsDraft', [
             'applyDraft',
         ]),
+        ...mapActions('list', [
+            'getElements',
+        ]),
         ...mapActions('gridDraft', [
             'removeDraft',
             'forceDraftsMutation',
         ]),
+        onSelectTabBarItem(index) {
```

Comment:
It's going to be called each time we choose vertical tab - each time we will call for data from API

Reply:
yes, but this behavior is correct, because we use 2 different list (attributes, system attributes) store and I must replace this information every time we change tab.

Reply:
What if we get 3rd tab? we are going to call it once again - the performance will drop instantly ( depending how many data comes from API ). Also not only data may have impact on those changes. We may see some artefacts without nullifying store after we change tab.